bigtop.mk

```diff
@@ -293,10 +293,10 @@ IMPALA_NAME=impala
 IMPALA_RELNOTES_NAME=Impala
 IMPALA_PKG_NAME=impala
 IMPALA_BASE_VERSION=2.2.0
-IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.3.0
+IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.4.0
 IMPALA_RELEASE_VERSION=1
```

Comment:
It's safe to increment the release number as it doesn't matter what the number is until you have a second release with the same version number. So if it solves a problem internally, I'm +1 to that change.